addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -202,6 +203,65 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+publicWidget.registry.cookies_bar = PopupWidget.extend({
+    selector: '#website_cookies_bar',
+    start() {
+        if (
+            this.el.querySelector(".o_cookies_popup")
```

Comment:
Isn't better to include this directly in the selector: `selector: '#website_cookies_bar:has(.o_cookies_popup)'` ?

Reply:
Probably, yes. Thanks, I always forget that it uses jQuery's selectors which includes `:has()`.

Reply:
I answered too fast. On second thought, it's actually the same reason as for the `PopupWidget.extend()`: to get the same base code as in 16.0+.